swan-lake/learn/testing-ballerina-code/writing-tests.md

```diff
@@ -60,14 +60,13 @@ Once the test module is imported, the following annotation can be used to write
 The function specified after the annotation is a test function. This annotation supports the following value fields.
 
 
-*   ***enable: {true&#124;false}*** - Enable/disable the test. The default value is `true`.
-*   ***before: "&lt;function name&gt;"*** - Name of the function to be run just before the test is run. The default value is `none`.
-*   ***after: "&lt;function name&gt;"*** - Name of the function to be run just after the test is run.
-*   ***dependsOn: ["&lt;function names>", …]*** - List of function names on which the test function depends. The
+*   ***enable: {true&#124;false}*** - Enable/disable the test. Default: true
+*   ***before: &lt;function name&gt;*** - The function to be run just before the test is run. Default: none
+*   ***after: &lt;function name&gt;*** - The function to be run just after the test is run. Default: none
+*   ***dependsOn: [&lt;function names>, …]*** - List of functions on which the test function depends. The
  order in
- which the comma-separated list appears has no prominence. In case there needs to be an order, define a sequence of test functions with one pointing to another based on the dependencies using the `dependsOn` parameter in each one’s config to control the order of the test execution.
-*   ***dataProvider: “&lt;function name>”*** - Specifies the name of the function that will be used to provide the value
- sets to execute the test against.
+ which the comma-separated list appears has no prominence. In case there needs to be an order, define a sequence of test functions with one pointing to another based on the dependencies using the `dependsOn` parameter in each one’s config in order to control the order of test execution.
```

Comment:
```suggestion
 which the comma-separated list appears has no prominence. In case there needs to be an order, define a sequence of test functions with one pointing to another based on the dependencies using the `dependsOn` parameter in each one’s config in order to control the order of test execution.
```
This sentence is too long. Need to break it down to multiple sentences.

Reply:
Made the sentence shorter and easier to read